nats-streaming-server.go

```diff
@@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
 	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
 	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
-	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
+	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
```

Comment:
+1, I think this would make it simpler to use and understand. One flag vs.
two.

On Wed, Jul 13, 2016 at 10:37 PM, Derek Collison <notifications@github.com>
wrote:

> In nats-streaming-server.go
> <https://github.com/nats-io/nats-streaming-server/pull/125#discussion_r70747489>
> :
>
> > @@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
> >  	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
> >  	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
> >  	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
> > -	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
> > +	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
>
> Why not assume embedded, but if you supply --server you present it with
> URL for the NATS server to connect to?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nats-io/nats-streaming-server/pull/125/files/a1d758fd084007e2f30e194bad2f7525e8236061..1b7ca0d88480c871cf6e334a647ed6194844fc21#r70747489>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGq4LL2CMLTof-INHtIijtYkb40fxma6ks5qVb0ZgaJpZM4JLmMr>
> .
>



-- 
*Larry McQueary* | Director, Messaging Technology
larry@apcera.com | @mcqueary | github.com/mcqueary